Tests fail on Windows

Looks like some sort of double-escaping issue.  Don't you hate this dumb platform?  I haven't really looked into it just yet, working on #33 first.

```
 1) bro preprocessing should create bundle:
     Uncaught AssertionError: expected '(function e(t,n,r){function s(o,u){if(!n[o]){if(!t[o]){var a=typeof require=="function"&&require;if(!u&&a)return a(o,!0);if(i)return i(o,!0);var f=new Error("Cannot find module \'"+o+"\'");throw f.code="MODULE_NOT_FOUND",f}var l=n[o]={exports:{}};t[o][0].call(l.exports,function(e){var n=t[o][1][e];return s(n?n:e)},l,l.exports,e,t,n,r)}return n[o].exports}var i=typeof require=="function"&&require;for(var o=0;o<r.length;o++)s(r[o]);return s})({"C:\\\\Code\\\\karma-bro\\\\test\\\\fixtures\\\\a.js":[function(require,module,exports){\n\'use strict\';\r\n\r\nmodule.exports = \'A\';\n},{}],"C:\\\\Code\\\\karma-bro\\\\test\\\\fixtures\\\\b.js":[function(require,module,exports){\n\'use strict\';\r\n\r\nvar a = require(\'./a\');\r\n\r\nmodule.exports = \'B\' + a;\n},{"./a":"C:\\\\Code\\\\karma-bro\\\\test\\\\fixtures\\\\a.js"}]},{},["C:\\\\Code\\\\karma-bro\\\\test\\\\fixtures\\\\b.js"]);\n' to include '"C:\\Code\\karma-bro\\test\\fixtures\\b.js"'
      at C:\Code\karma-bro\test\spec\pluginSpec.js:267:39
      at fileProcessed (C:\Code\karma-bro\test\spec\pluginSpec.js:73:7)
      at C:\Code\karma-bro\test\spec\pluginSpec.js:79:7
      at C:\Code\karma-bro\lib\bro.js:270:9
      at Browserify.<anonymous> (C:\Code\karma-bro\lib\bro.js:244:9)
      at Browserify.g (events.js:199:16)
      at Browserify.emit (events.js:129:20)
      at C:\Code\karma-bro\lib\bro.js:225:11
      at C:\Code\karma-bro\node_modules\browserify\index.js:664:13
      at ConcatStream.<anonymous> (C:\Code\karma-bro\node_modules\browserify\node_modules\concat-stream\index.js:32:43)

  2) bro preprocessing should pass through on updates:
     Uncaught AssertionError: expected '(function e(t,n,r){function s(o,u){if(!n[o]){if(!t[o]){var a=typeof require=="function"&&require;if(!u&&a)return a(o,!0);if(i)return i(o,!0);var f=new Error("Cannot find module \'"+o+"\'");throw f.code="MODULE_NOT_FOUND",f}var l=n[o]={exports:{}};t[o][0].call(l.exports,function(e){var n=t[o][1][e];return s(n?n:e)},l,l.exports,e,t,n,r)}return n[o].exports}var i=typeof require=="function"&&require;for(var o=0;o<r.length;o++)s(r[o]);return s})({"C:\\\\Code\\\\karma-bro\\\\test\\\\fixtures\\\\a.js":[function(require,module,exports){\n\'use strict\';\r\n\r\nmodule.exports = \'A\';\n},{}],"C:\\\\Code\\\\karma-bro\\\\test\\\\fixtures\\\\b.js":[function(require,module,exports){\n\'use strict\';\r\n\r\nvar a = require(\'./a\');\r\n\r\nmodule.exports = \'B\' + a;\n},{"./a":"C:\\\\Code\\\\karma-bro\\\\test\\\\fixtures\\\\a.js"}]},{},["C:\\\\Code\\\\karma-bro\\\\test\\\\fixtures\\\\b.js"]);\n' to include '"C:\\Code\\karma-bro\\test\\fixtures\\b.js"'
      at C:\Code\karma-bro\test\spec\pluginSpec.js:295:41
      at fileProcessed (C:\Code\karma-bro\test\spec\pluginSpec.js:73:7)
      at C:\Code\karma-bro\test\spec\pluginSpec.js:79:7
      at C:\Code\karma-bro\lib\bro.js:270:9
      at Browserify.<anonymous> (C:\Code\karma-bro\lib\bro.js:244:9)
      at Browserify.g (events.js:199:16)
      at Browserify.emit (events.js:129:20)
      at C:\Code\karma-bro\lib\bro.js:225:11
      at C:\Code\karma-bro\node_modules\browserify\index.js:664:13
      at ConcatStream.<anonymous> (C:\Code\karma-bro\node_modules\browserify\node_modules\concat-stream\index.js:32:43)
```

--- a/package.json
+++ b/package.json
@@ -40,7 +40,8 @@
     "grunt-release": "~0.7.0",
     "karma": "^0.12.19",
     "load-grunt-tasks": "~0.4.0",
-    "tsify": "^0.5.0"
+    "tsify": "^0.5.0",
+    "browser-unpack": "~1.0.0"
   },
   "peerDependencies": {
     "karma": ">=0.10"

--- a/test/fixtures/c.js
+++ b/test/fixtures/c.js
@@ -1,0 +1,5 @@
+'use strict';
+
+var a = require('./a');
+
+module.exports = 'C' + a;

--- a/test/spec/pluginSpec.js
+++ b/test/spec/pluginSpec.js
@@ -15,6 +15,8 @@ var chai = require('chai');
 var path = require('path'),
     fs = require('fs');
 
+var unpack = require('browser-unpack');
+
 
 function delay(fn, time) {
   setTimeout(fn, time || 205);
@@ -254,20 +256,28 @@ describe('bro', function() {
       var plugin = createPlugin();
 
       var bundleFile = createFile(bundle.location);
-      var testFile = createFile('test/fixtures/b.js');
+      var testFileB = createFile('test/fixtures/b.js');
+      var testFileC = createFile('test/fixtures/c.js');
 
       // when
-      plugin.preprocess(bundleFile, [ testFile ], function(order) {
+      plugin.preprocess(bundleFile, [ testFileB, testFileC ], function(order) {
 
         // then
         // resolve order: bundle, testFileStubs ...
-        expect(order).to.eql([ bundleFile, testFile ]);
+        expect(order).to.eql([ bundleFile, testFileB, testFileC ]);
 
         // bundle got created
-        expect(bundleFile.bundled).to.contain('"' + path.resolve('test/fixtures/b.js') + '"');
+        var entryPointFiles = unpack(bundleFile.bundled)
+          .filter(function (row) { return row.entry; })
+          .map(function (row) { return row.id; });
+        expect(entryPointFiles).to.eql([
+          path.resolve('test/fixtures/b.js'),
+          path.resolve('test/fixtures/c.js')
+        ]);
 
         // test file stub got created
-        expect(testFile.bundled).to.eql('/* bundled */');
+        expect(testFileB.bundled).to.eql('/* bundled */');
+        expect(testFileC.bundled).to.eql('/* bundled */');
 
         done();
       });
@@ -292,7 +302,10 @@ describe('bro', function() {
           // then
 
           // bundle got passed through
-          expect(bundleFile.bundled).to.contain('"' + path.resolve('test/fixtures/b.js') + '"');
+          var entryPointFiles = unpack(bundleFile.bundled)
+            .filter(function (row) { return row.entry; })
+            .map(function (row) { return row.id; });
+          expect(entryPointFiles).to.eql([ path.resolve('test/fixtures/b.js') ]);
 
           // test file got regenerated
           expect(testFile.bundled).to.eql('/* bundled */');

